### DISCUSSION TITLE
How to Reset Brackets Web Version back to its original state

### DISCUSSION THREAD

**Discussion by williadn58** [justinrainbow/json-schema#749] [General]
I deleted some things from brackets.io web version and now nothing works.  How do I reset the brackets.io web version back to its original state?  When I open a new browser, its holding on to the previous web version where nothing works because I mucked it up.

**Comment by DannyvdSluijs** [comment]
@williadn58 thanks for your report. But I’m struggling to see what is the issue and how this relates to this repo. As far as I’m am aware this repo has no relation to brackets.io

Could you elaborate on your issue and check if the report is done at the correct repo?
